src/pseudopeople/utilities.py

```diff
@@ -28,7 +28,7 @@ def vectorized_choice(
     randomness_stream: RandomnessStream,
     weights: Union[list, pd.Series] = None,
     additional_key: Any = None,
-) -> pd.Series:
+) -> np.ndarray:
```

Comment:
This was wrong.

Reply:
Oops, I probably forgot to update this when I optimized the function!